package/MDAnalysis/coordinates/IMD.py

```diff
@@ -0,0 +1,152 @@
+"""
+MDAnalysis IMDReader
+^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: IMDReader
+   :members:
+   :inherited-members:
+
+"""
+
+from MDAnalysis.coordinates import core
+from MDAnalysis.lib.util import store_init_arguments
+from MDAnalysis.coordinates.util import parse_host_port
+from MDAnalysis.coordinates.base import StreamReaderBase
+
+try:
+    import imdclient
+    from imdclient.IMDClient import IMDClient
+except ImportError:
+    HAS_IMDCLIENT = False
+
+    # Allow building doucmnetation without imdclient
+    import types
+
+    class MockIMDClient:
+        pass
+    imdclient = types.ModuleType("imdclient")
+    imdclient.IMDClient = MockIMDClient
+
+else:
+    HAS_IMDCLIENT = True
+
+import logging
+
+logger = logging.getLogger("imdclient.IMDClient")
+
+
+class IMDReader(StreamReaderBase):
```

Comment:
I think I'm getting confused about which PR is for which thing. @orbeckst given our discussion earlier this week, and your comment above which I take to be "IMDClient is still in flux", does it not make sense for the IMDReader to exist upstream and then just import it here?

Reply:
We have to split IMDReader from imdclient and make a version of imdclient without IMDReader (which is in the works https://github.com/Becksteinlab/imdclient/pull/54 ). At the same time we are moving what was split off into coordinates.IMD. 

Amru is working on both at the moment.

The way  IMDReader depends on imdclient is not the problem, and imdclient itself is also pretty stable, it's just that the tests for imdclient have made use of a lot of MDAnalysis/IMDReader for convenience, and we now have to rewrite some of these tests to use bare-bones python.

Reply:
This is done now!